server/api.js

```diff
@@ -8,4 +8,29 @@ router.get("/", (_, res) => {
 	res.json({ message: "Hello, world!" });
 });
 
+router.post("/login",async (req , res) => {
+	const { email, password } = req.body;
+	try{
+		// Check if email exists in the database
+		const query = "SELECT * FROM users WHERE email = $1";
+		const dbResponse = await pool.query(query, [email]);
+		if(dbResponse.row.length === 0){
+			// If email doesn't exist
+			return res.status(404).json({ message : "Email does not exist! if you don't have an account sign up first!" , success : false });
```

Comment:
Returns an error code and let the frontend build the error message. Maybe something like INVALID_LOGIN and make the response a 401